HACKING

```diff
@@ -0,0 +1,26 @@
+ MODIFYING OPENSSL SOURCE
+ ------------------------
+ This document describes the way to add custom modifications to OpenSSL sources.
+ 
+ If you are adding new public functions to the custom library build, you need to
+ either add a prototype in one of the existing OpenSSL header files;
+ or provide a new header file and edit Configurations/unix-Makefile.tmpl to pick up that file.
+ 
+ After that perform the following steps:
+ 
+    ./configure '-Werror --strict-warnings' [your-options]
```

Comment:
This still works without the single quotes, best to be rid of them.

Reply:
Also, it should be either config or Configure, but not configure

Reply:
Now, I feel like I had selective blindness.  Best make it:
```
./config -Werror --strict-warnings [your-options]
```

Reply:
Fixed.